contributingGuides/KSv2.md

```diff
@@ -0,0 +1,67 @@
+# KSv2
+
+In Expensify, we use a prioritization system called called "Kernel Scheduler v2" (KSv2) to help us keep aligned on how frequently GitHub issues should be worked on, depending on their time priority:
+
+**Hourly**: If a Github issue has the `Hourly` label, the assignee should update it *at least* once an hour
+
+**Daily**: If a Github issue has the `Daily` label, the assignee should update it *at least* once a day
+
+**Weekly**: If a Github issue has the `Weekly` label, the assignee should update it *at least* once a week
+
+**Monthly**: If a Github issue has the `Monthly` label, the assignee should update it *at least* once a month.
+
+## Dashboard
+
+To help surface the issues and PRs that need the most `#urgency`, we've built a custom extension to use in GitHub. It looks like this:
+
+<img src="https://user-images.githubusercontent.com/6829422/213875977-8ff4cf19-7690-4203-ae13-a8da259be7d0.png" />
+
+Once you have the extension installed, you can access it by going to https://github.com/Expensify/Expensify#k2 or clicking on the K2 tab at the top of a repo (between pull requests and GH actions).
```

Comment:
```suggestion
Once you have the extension installed, you can access it by going to https://github.com/Expensify/Expensify#k2 or clicking on the K2 tab at the top of a repo (between pull requests and GH actions). You'll have to create a Personal Access Token in GitHub ([here](https://github.com/settings/tokens)) and enter that in the K2 dashboard the first time you open it, so that your K2 extension can automatically pull data about your GitHub account.
```

Reply:
Or maybe this can go under "Installation" below?

Reply:
I kinda feel like this should be self-explanatory in the UI itself, and if it's not, let's make it that way soon

Reply:
but I'm OK having this for now (if it's not self-explanatory currently)

Reply:
I think it's self explanatory that a PAT needs to be created, but how to do that & why is not necessarily self explanatory yet